### PR TITLE
DOC: Terms update in documentation

### PR DESCRIPTION
**Pull request by Shubham9t9**
# Description

Changes Git material to Git Repository  and Docker Registry to Container registry in documentation.

Fixes # (issue)

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# Checklist:

* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [X] Does this PR require documentation updates?
* [X] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have tested it for all user roles

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 ## User Guide
 
 * [Creating Application](user-guide/creating-application/README.md)
-  * [Git Material](user-guide/creating-application/git-material.md)
+  * [Git Repository](user-guide/creating-application/git-repository.md)
```
Comment (prakarsh-dt) [inline docs/SUMMARY.md:18]: Do not rename the files as these are linked from within the product and can affect SEO too as these links are already crawled on google. Leave the filenames as they are unless we are able to find a way for implementing 301 redirects

File: docs/SUMMARY.md [hunk 2]
```diff
@@ -42,7 +42,7 @@
 * [Global Configurations](user-guide/global-configurations/README.md)
   * [GitOps](user-guide/global-configurations/gitops.md)
   * [Git Accounts](user-guide/global-configurations/git-accounts.md)
-  * [Docker Registries](user-guide/global-configurations/docker-registries.md)
+  * [Container Registries](user-guide/global-configurations/container-registries.md)
```
Comment (prakarsh-dt) [inline docs/SUMMARY.md:45]: Don't rename the file

File: docs/user-guide/creating-application/environment-overrides.md [hunk 3]
```diff
@@ -2,7 +2,8 @@
 
 You will see all your environments associated with an application under the `Environment Overrides` section.
 
-![](../../.gitbook/assets/env_ride%20%283%29.jpg)
+![](../../user-guide/creating-application/images/environment-override.png)
```
Comment (prakarsh-dt) [inline docs/user-guide/creating-application/environment-overrides.md:5]: Use images in jpg

File: docs/SUMMARY.md (hunk 2 above)
Comment (Shubham9t9) [reply docs/SUMMARY.md:45]: Done

File: docs/user-guide/creating-application/environment-overrides.md (hunk 3 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/environment-overrides.md]: Done

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**